site/content/docs/5.1/forms/quantity-selector.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: docs
+title: Quantity selector
+description: Form element used to select a number.
+group: forms
+toc: true
+---
+
+## Examples
+
+Quantity selector is a form element used to select a number. The default version is the small version. To use the large version, use the contextual class `.quantity-selector-l`.
+
+You can specify a default value in the `value` attribute of your input.
+
+Value will vary between the values define in the `min` and `max` attributes (negatives values are allowed).
+
+The custom `data-bs-round` attribute will help you to define the number of digits to appear after the decimal point.
+
+{{< example >}}
+<div class="quantity-selector mb-3">
```

Comment:
Why the `.quantity-selector` is on an extra `<div>`?
IMO, we should be able to do something like:

```
<form>
  <div class="mb-3">
    <label for="inputQuantitySelector1" class="form-label">Email address</label>
    <div class="input-group quantity-selector">
      <input type="number" id="inputQuantitySelector1" class="form-control" aria-live="polite" data-bs-step="counter" name="quantity" title="quantity" value="0" min="0" max="10" step="1" data-bs-round="0" aria-label="Quantity selector">
      <button type="button" class="btn btn-icon btn-secondary btn-sm"  aria-describedby="inputQuantitySelector1" data-bs-step="down">
        <span class="visually-hidden">Step down</span>
      </button>
      <button type="button" class="btn btn-icon btn-secondary btn-sm"  aria-describedby="inputQuantitySelector1" data-bs-step="up">
        <span class="visually-hidden">Step up</span>
      </button>
    </div>
    <div id="quantitySelectorHelp" class="form-text">Lorem ipsum.</div>
  </div>
  <button type="submit" class="btn btn-primary">Submit</button>
</form>
```

Reply:
Done